### PR TITLE
Assess application tasklist

### PR DESCRIPTION
**Pull request by benbaumann95**
### Description of change

- Create new controller / routes for assessment tasks 
- Move assess against legislation into this tasklist and extract logic into presenter
- Add placeholder "Check consistency" section in tasklist for upcoming work

- In following PR's we will move some more assessment flows into this new tasklist

### Story Link

https://trello.com/c/rXuoJxk1/1168-create-an-assess-the-application-tasklist-page

Screenshot 2022-09-02 at 11 13 58

### CODE REVIEW

File: app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module AssessmentTasks
+  extend ActiveSupport::Concern
+
+  class AssessAgainstLegislationPresenter < PlanningApplicationPresenter
```
Comment (allbecauseyoutoldmeso) [inline app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb:6]: Does this need to inherit from `PlanningApplicationPresenter`?

File: app/views/policy_classes/part.html.erb [hunk 2]
```diff
@@ -48,7 +48,7 @@
 
       <div class="govuk-button-group">
         <%= form.submit "Continue", class: "govuk-button" %>
-        <%= link_to "Back", @planning_application, class: "govuk-button govuk-button--secondary" %>
+        <%= link_to "Back", :back, class: "govuk-button govuk-button--secondary" %>
```
Comment (allbecauseyoutoldmeso) [inline app/views/policy_classes/part.html.erb:51]: 🙌

**Review by allbecauseyoutoldmeso — APPROVED** [review]
Nicely done!

Love the moving-into-presenters stuff.

Please merge this soon, or I'm going to have SO MANY merge conflicts with the policy classes refactor 🤣

**Review by EGiataganas — APPROVED** [review]
Great work

File: app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb (hunk 1 above)
Comment (benbaumann95) [reply app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb:6]: Yeah I think so to call `super(template, planning_application)`

**Pull request merged**